tests/bluetooth/bsim/nrf_auraconfig/tester/README.txt

```diff
@@ -0,0 +1,7 @@
+# nRF Auraconfig Tester
+
+This test code will set up a broadcast sink to receive and parse the advertisement packets from the nRF Auraconfig using BabbleSim and verify that the result matches the test vector supplied. It is not intended for use in production code.
+Compared to other tests, these run as integration tests involving both the controller, host and application layer.
+
+These tests are run as part of nRF Connect SDK CI with specific configurations.
+They should not be treated as a reference for general use, as they are designed for this CI only and fault injection purposes.
```

Comment:
```suggestion
They should not be treated as a reference for general use, as they are designed for this CI only and fault injection purposes.

## Requirements

The test supports the `nrf5340_audio_dk`.
```

Reply:
It doesn't though. This test is strictly BabbleSim, so I can change out the nrf5340_audio_dk with the corresponding bsim-board